dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/trackedentity/TrackedEntitiesExportControllerTest.java

```diff
@@ -513,8 +512,8 @@ void getTrackedEntityCsvById() {
     assertTrue(response.header("content-disposition").contains("filename=trackedEntity.csv"));
     assertStartsWith(
         """
-trackedEntity,trackedEntityType,createdAt,createdAtClient,updatedAt,updatedAtClient,orgUnit,inactive,deleted,potentialDuplicate,geometry,latitude,longitude,storedBy,createdBy,updatedBy,attrCreatedAt,attrUpdatedAt,attribute,displayName,value,valueType
-""",
+    trackedEntity,trackedEntityType,createdAt,createdAtClient,updatedAt,updatedAtClient,orgUnit,inactive,deleted,potentialDuplicate,geometry,latitude,longitude,storedBy,createdBy,updatedBy,attrCreatedAt,attrUpdatedAt,attribute,displayName,value,valueType
```

Comment:
is this added whitespace now in the csv 🤔

Reply:
<img width="1119" alt="Screenshot 2025-03-04 at 09 00 08" src="https://github.com/user-attachments/assets/952cdc85-c61a-4a09-8e40-c8d1e323f163" />

I didn't change anything here and git blame is weird 🤔